### PR TITLE
Use example-specific web-pages for Java examples

### PR DESCRIPTION
**Pull request by Dmytro-Fedoryshyn**
In this changeset, we have utilized example-specific web-pages for our examples.

Issue: TeamDev-IP/JxBrowser/issues/4091

### CODE REVIEW

**Comment by Dmytro-Fedoryshyn** [comment]
26 tests in Java and 19 tests in Kotlin were using [Google](google.com) or [TeamDev](teamdev.com). We have substituted them with either [HTML5 Test](https://html5test.teamdev.com/) for simple examles or [Dymamic WebGL cubemap](https://webglsamples.org/dynamic-cubemap/dynamic-cubemap.html) for examples, where it's important to demonstrate the capabilities of JxBrowser.

File: examples/src/main/java/com/teamdev/jxbrowser/examples/DefaultZoomLevel.java [hunk 1]
```diff
@@ -61,6 +61,7 @@ public void windowClosing(WindowEvent e) {
 
         engine.zoomLevels().defaultLevel(ZoomLevel.P_300);
 
-        browser.navigation().loadUrl("https://www.google.com");
+        browser.mainFrame()
+               .ifPresent(mainFrame -> mainFrame.loadHtml("https://html5test.teamdev.com"));
```
Comment (vladimir-ikryanov) [inline examples/src/main/java/com/teamdev/jxbrowser/examples/DefaultZoomLevel.java:65]: It's wrong. Just replace URL.

File: examples/src/main/java/com/teamdev/jxbrowser/examples/DisableZoom.java [hunk 2]
```diff
@@ -59,7 +59,8 @@ public void windowClosing(WindowEvent e) {
             frame.setVisible(true);
         });
 
-        browser.navigation().loadUrlAndWait("https://www.google.com");
+        browser.mainFrame()
+               .ifPresent(mainFrame -> mainFrame.loadHtml("https://html5test.teamdev.com"));
```
Comment (vladimir-ikryanov) [inline examples/src/main/java/com/teamdev/jxbrowser/examples/DisableZoom.java:63]: It's wrong. Just replace URL.

File: examples/src/main/java/com/teamdev/jxbrowser/examples/PrintFromJava.java [hunk 3]
```diff
@@ -65,7 +64,8 @@ public void windowClosing(WindowEvent e) {
             frame.setLocationRelativeTo(null);
             frame.setVisible(true);
 
-            browser.navigation().loadUrl("https://www.google.com");
+            browser.navigation()
+                   .loadUrl("https://webglsamples.org/dynamic-cubemap/dynamic-cubemap.html");
```
Comment (vladimir-ikryanov) [inline examples/src/main/java/com/teamdev/jxbrowser/examples/PrintFromJava.java:68]: For Print Preview example it's better to load a text page. Let's load https://en.wikipedia.org/wiki/Printing

File: examples/src/main/java/com/teamdev/jxbrowser/examples/PrintSettings.java [hunk 4]
```diff
@@ -77,7 +77,8 @@ public static void main(String[] args) {
             // #enddocfragment "Proceed"
         });
         // #enddocfragment "Callback"
-        browser.navigation().loadUrlAndWait("https://google.com");
+        browser.navigation()
+               .loadUrlAndWait("https://webglsamples.org/dynamic-cubemap/dynamic-cubemap.html");
```
Comment (vladimir-ikryanov) [inline examples/src/main/java/com/teamdev/jxbrowser/examples/PrintSettings.java:81]: For Print Preview example it's better to load a text page. Let's load https://en.wikipedia.org/wiki/Printing

File: examples/src/main/java/com/teamdev/jxbrowser/examples/PrintToPdf.java [hunk 5]
```diff
@@ -63,7 +64,8 @@ public static void main(String[] args) {
             });
             tell.proceed(pdfPrinter);
         });
-        browser.navigation().loadUrlAndWait("https://google.com");
+        browser.navigation()
+               .loadUrlAndWait("https://webglsamples.org/dynamic-cubemap/dynamic-cubemap.html");
```
Comment (vladimir-ikryanov) [inline examples/src/main/java/com/teamdev/jxbrowser/examples/PrintToPdf.java:68]: For Print Preview example it's better to load a text page. Let's load https://en.wikipedia.org/wiki/Printing

File: examples/src/main/java/com/teamdev/jxbrowser/examples/SaveWebPage.java [hunk 6]
```diff
@@ -61,7 +61,8 @@ public void windowClosing(WindowEvent e) {
             frame.setVisible(true);
         });
 
-        browser.navigation().loadUrlAndWait("https://www.google.com");
+        browser.navigation()
+               .loadUrlAndWait("https://webglsamples.org/dynamic-cubemap/dynamic-cubemap.html");
```
Comment (vladimir-ikryanov) [inline examples/src/main/java/com/teamdev/jxbrowser/examples/SaveWebPage.java:65]: It's not a good idea to save a web page with WebGL content. It should be the html5test page.

File: examples/src/main/java/com/teamdev/jxbrowser/examples/javafx/BitmapToJavaFxImage.java [hunk 7]
```diff
@@ -47,7 +47,8 @@ public static void main(String[] args) throws IOException {
             browser.resize(1024, 768);
 
             // Load the required web page and wait until it is loaded completely
-            browser.navigation().loadUrlAndWait("https://www.google.com");
+            browser.navigation()
+                   .loadUrlAndWait("https://webglsamples.org/dynamic-cubemap/dynamic-cubemap.html");
```
Comment (vladimir-ikryanov) [inline examples/src/main/java/com/teamdev/jxbrowser/examples/javafx/BitmapToJavaFxImage.java:51]: Let's use html5test here.

File: examples/src/main/java/com/teamdev/jxbrowser/examples/swing/BitmapToSwingImage.java [hunk 8]
```diff
@@ -45,7 +45,8 @@ public static void main(String[] args) throws IOException {
             browser.resize(1024, 768);
 
             // Load the required web page and wait until it is loaded completely
-            browser.navigation().loadUrlAndWait("https://www.google.com");
+            browser.navigation()
+                   .loadUrlAndWait("https://webglsamples.org/dynamic-cubemap/dynamic-cubemap.html");
```
Comment (vladimir-ikryanov) [inline examples/src/main/java/com/teamdev/jxbrowser/examples/swing/BitmapToSwingImage.java:49]: Let's use html5test here.

File: examples/src/main/java/com/teamdev/jxbrowser/examples/swing/BrowserViewInJTabbedPane.java [hunk 9]
```diff
@@ -62,7 +62,10 @@ public void windowClosing(WindowEvent e) {
             frame.setVisible(true);
         });
 
-        browserOne.navigation().loadUrl("https://www.google.com");
-        browserTwo.navigation().loadUrl("https://www.teamdev.com");
+        browserOne.mainFrame()
+                  .ifPresent(mainFrame -> mainFrame.loadHtml("https://html5test.teamdev.com/"));
+        browserTwo.mainFrame()
+                  .ifPresent(mainFrame -> mainFrame.loadHtml(
+                          "https://webglsamples.org/dynamic-cubemap/dynamic-cubemap.html"));
```
Comment (vladimir-ikryanov) [inline examples/src/main/java/com/teamdev/jxbrowser/examples/swing/BrowserViewInJTabbedPane.java:69]: Wrong. Just load the URLs.

File: examples/src/main/java/com/teamdev/jxbrowser/examples/swt/BitmapToSwtImage.java [hunk 10]
```diff
@@ -47,7 +47,8 @@ public static void main(String[] args) {
             browser.resize(1024, 768);
 
             // Load the required web page and wait until it is loaded completely
-            browser.navigation().loadUrlAndWait("https://www.google.com");
+            browser.navigation()
+                   .loadUrlAndWait("https://webglsamples.org/dynamic-cubemap/dynamic-cubemap.html");
```
Comment (vladimir-ikryanov) [inline examples/src/main/java/com/teamdev/jxbrowser/examples/swt/BitmapToSwtImage.java:51]: Let's use html5test page here.

File: examples/src/main/kotlin/com/teamdev/jxbrowser/examples/PrintFromKotlin.kt [hunk 11]
```diff
@@ -49,6 +49,6 @@ fun main() = singleWindowApplication(title = "Print from Java") {
     }
 
     LaunchedEffect(Unit) {
-        browser.navigation.loadUrl("https://www.google.com")
+        browser.navigation.loadUrl("https://webglsamples.org/dynamic-cubemap/dynamic-cubemap.html")
```
Comment (vladimir-ikryanov) [inline examples/src/main/kotlin/com/teamdev/jxbrowser/examples/PrintFromKotlin.kt:52]: In all print samples please use https://en.wikipedia.org/wiki/Printing

Comment (vladimir-ikryanov) [reply examples/src/main/kotlin/com/teamdev/jxbrowser/examples/PrintFromKotlin.kt:52]: Repeat my notes for Java code for the following Kotlin examples.

**Review by vladimir-ikryanov — CHANGES_REQUESTED** [review]

File: examples/src/main/java/com/teamdev/jxbrowser/examples/PrintToPdf.java [hunk 12]
```diff
@@ -49,7 +50,7 @@ public static void main(String[] args) {
                     params.printers().pdfPrinter();
             PrintJob<HtmlSettings> printJob = pdfPrinter.printJob();
             printJob.settings()
-                    .pdfFilePath(Paths.get("google.pdf").toAbsolutePath())
+                    .pdfFilePath(Paths.get("dynamic-cubemap.pdf").toAbsolutePath())
```
Comment (vladimir-ikryanov) [inline examples/src/main/java/com/teamdev/jxbrowser/examples/PrintToPdf.java:53]: Update the pdf file name.

**Review by vladimir-ikryanov — CHANGES_REQUESTED** [review]

**Pull request merged**